source/how-tos/boot-ubuntu.rst

```diff
@@ -5,26 +5,10 @@ Boot Ubuntu
 
 |SPN| can boot Ubuntu Linux using |SPN| with default OS loader. This page provides a step-by-step guide how to do so.
 
-The following steps have been verified with Ubuntu Linux 18.04 LTS (64-bit) on |UP2| board.

```

Comment:
Maybe good to leave it has been verified  with Ubuntu Linux 18.04 LTS (64-bit)

Reply:
Done